### PR TITLE
Refs[TASK][36446] show ingredient + alphabets

### PR DESCRIPTION
**Pull request by VietKFC**
## Related Tickets
- [#Task 36446: Content](https://edu-redmine.sun-asterisk.vn/issues/36446)
## WHAT
-
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/ingredient_include.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:layout_marginTop="@dimen/dp_4"
+                android:text="@string/text_rum"
+                android:textColor="@android:color/black"
+                android:textSize="@dimen/sp_16" />
+        </LinearLayout>
+
+        <LinearLayout
```
Comment (thonc-2135) [inline app/src/main/res/layout/ingredient_include.xml:112]: dùng nhiều linear thế, sao k dùng chain

File: app/src/main/res/layout/alphabet_item.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="@dimen/dp_40"
+    android:layout_height="@dimen/dp_40"
+    android:layout_marginTop="@dimen/dp_4"
+    app:cardCornerRadius="@dimen/dp_4">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/border_background">
```
Comment (thonc-2135) [inline app/src/main/res/layout/alphabet_item.xml:12]: bỏ constaint r vút back ground lên cardview đk k

Comment (VietKFC) [reply app/src/main/res/layout/alphabet_item.xml:12]: Em thấy thằng cardview nó ko hiện cái background này ạ

File: app/src/main/res/layout/ingredient_include.xml (hunk 1 above)
Comment (VietKFC) [reply app/src/main/res/layout/ingredient_include.xml:112]: Em vừa bỏ linear layout  bên ngoài rồi ạ , còn linear ben trong để e nhóm imageview + text ạ

File: app/src/main/res/layout/ingredient_include.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            android:id="@+id/linear_rum"
+            app:layout_constraintRight_toLeftOf="@id/linear_tequila"
+            app:layout_constraintTop_toTopOf="@id/linear_gin"
+            app:layout_constraintBottom_toBottomOf="@id/linear_gin"
+            app:layout_constraintLeft_toRightOf="@id/linear_gin"
+            android:orientation="vertical">
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="100dp"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/ingredient_include.xml]: hardcode

File: app/src/main/res/layout/ingredient_include.xml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            app:layout_constraintTop_toTopOf="@id/linear_rum"
+            app:layout_constraintBottom_toBottomOf="@id/linear_rum"
+            app:layout_constraintLeft_toRightOf="@id/linear_rum"
+            app:layout_constraintRight_toRightOf="parent"
+            android:layout_marginEnd="@dimen/dp_20"
+            android:orientation="vertical">
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="100dp"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/ingredient_include.xml]: hardcode

**Pull request merged**